resources/views/hardware/labels.blade.php

```diff
@@ -105,12 +109,19 @@
 	<?php $count++; ?>
   <div class="label"{!!  ($count % $settings->labels_per_page == 0) ? ' style="margin-bottom: 0px;"' : '' !!}>
 
+      @if (($settings->labels_display_company_logo=='1') && ($asset->company->image))
```

Comment:
Warning: this leads to an error if asset has no company. 
Condition should be:
 @if (($settings->labels_display_company_logo=='1') && ($asset->company) && ($asset->company->image))